metabase/metabase
# Fix editable text issues

Epic https://github.com/metabase/metabase/issues/22826

Fixes:
- Makes it impossible to remove the question name for a saved question
- Fixes an issue with the placeholder when removing contents of the textarea

How to test:
- Make sure it's possible to edit the question's title and description

<img width="478" alt="Screenshot 2022-06-24 at 14 34 27" src="https://user-images.githubusercontent.com/8542534/175526733-a05c0feb-c340-486e-be21-852362421251.png">
